tensorzero-internal/src/clickhouse/migration_manager/migrations/migration_0020.rs

```diff
@@ -107,49 +128,94 @@ impl Migration for Migration0020<'_> {
             .duration_since(std::time::UNIX_EPOCH)
             .map_err(|e| {
                 Error::new(ErrorDetails::ClickHouseMigration {
-                    id: "0020".to_string(),
+                    id: MIGRATION_ID.to_string(),
```

Comment:
Let's compute `view_timestamp`after the table exchanges to ensure we have sufficient time.